cobalt/black_box_tests/tests/scroll_with_none_pointer_events.py

```diff
@@ -0,0 +1,78 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for scroll containers."""
+
+import logging
+import time
+import traceback
+
+from cobalt.black_box_tests import black_box_tests
+from cobalt.black_box_tests.threaded_web_server import ThreadedWebServer
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.common import keys
+
+# Time to sleep after a mouse move to give the device time to process it and
+# to avoid collapsing with subsequent move events.
+_SLEEP_AFTER_MOVE_TIME = 0.5
+
+
+def find_element_by_id(runner, id_selector):
+  return runner.webdriver.find_elements_by_css_selector('#' + id_selector)[0]
+
+
+class PointerTest(black_box_tests.BlackBoxTestCase):
+  """Tests pointer and mouse event."""
+
+  def test_pointer_events(self):
+    try:
+      with ThreadedWebServer(
+          binding_address=self.GetBindingAddress()) as server:
+        url = server.GetURL(
+            file_name='testdata/scroll_with_none_pointer_events.html')
+
+        with self.CreateCobaltRunner(url=url) as runner:
+          logging.info('JS Test Setup WaitForJSTestsSetup')
+          runner.WaitForJSTestsSetup()
+          logging.info('JS Test Setup')
+          self.assertTrue(runner.webdriver)
+
+          row = find_element_by_id(runner, 'row')
+          # Perform mouse actions with ActionChains.
+          #   https://www.selenium.dev/selenium/docs/api/py/webdriver/selenium.webdriver.common.action_chains.html#module-selenium.webdriver.common.action_chains  # pylint: disable=line-too-long
+          actions = ActionChains(runner.webdriver)
+
+          # No scrolling when pointer-events:none
+          actions.click_and_hold(row).pause(_SLEEP_AFTER_MOVE_TIME)
+          actions.move_by_offset(-100, 0)
+          actions.release()
+          actions.perform()
+          runner.SendKeys(keys.Keys.NUMPAD0)
+
+          # No scrolling if swipe starts with pointer-events:none,
+          # even if pointer-events is enabled during the swipe
+          # runner.SendKeys(keys.Keys.NUMPAD1)
+          actions2 = ActionChains(runner.webdriver)
+          actions2.click_and_hold(row).pause(_SLEEP_AFTER_MOVE_TIME)
+          actions2.send_keys(keys.Keys.NUMPAD1).pause(_SLEEP_AFTER_MOVE_TIME)
+          actions2.move_by_offset(-100, 0)
+          actions2.release()
+          actions2.perform()
+          runner.SendKeys(keys.Keys.NUMPAD0)
+          self.assertTrue(runner.JSTestsSucceeded())
+    except:  # pylint: disable=bare-except
```

Comment:
Kinda unrelated to this PR, but I'm curious if you know why we catch the exception here instead of just letting the process throw it? It seems like something we do throughout bb tests so should be something that happens at a higher level (by the test runner)

Reply:
I'm being lazy and just copying everything from scroll.py, which further copys pointer_tests.py? There are only a few tests do this way, so not sure if it's just some old issue, and no longer exists. I can remove this to see if it just works.

Reply:
No need to do much here, was just curious if you knew

Reply:
There are many tests that don't have this try/catch logic. Removed and nothing happened. If we find it's flaky we can add this back at any time.